src/api/views.py

```diff
@@ -83,58 +84,37 @@ def validate(request):
     elif request.method == 'POST':
         """ Return validate tool result on the post file"""
         serializer = ValidateSerializer(data=request.data)
+        
         if serializer.is_valid():
-            if (jpype.isJVMStarted()==0):
-                """ If JVM not already started, start it, attach a Thread and start processing the request """
-                classpath =settings.JAR_ABSOLUTE_PATH
-                jpype.startJVM(jpype.getDefaultJVMPath(),"-ea","-Djava.class.path=%s"%classpath)
-            """Attach a Thread and start processing the request """
-            jpype.attachThreadToJVM()
-            package = jpype.JPackage("org.spdx.tools")
-            verifyclass = package.Verify
-            query = ValidateFileUpload.objects.create(
-                owner=request.user,
-                file=request.data.get('file')
-            )
+            initialise_jpype()
+            output = license_validate_helper(request)
+            jpype.detachThreadFromJVM()
+
+            httpstatus = output.get('status', None)
+            context_dict = output.get('context', None)
+            response = output.get('response', None)
+            message = output.get('message', None)
```

Comment:
Have a common function for this too, as this conversion if needed in every api

Reply:
Sure. I was thinking the same. It would reduce code redundancy as this piece of code is used both in app and api.